### ISSUE TITLE
5.4

### ISSUE THREAD

**Issue by CharlieRoss2506** [CharlieRoss2506/Group-Project#13]

**Comment by jamesgould-a** [comment]
Im getting a Syntax-error for the %%HTML part of 5.4 that I'm unsure how to resolve, its specifically for the second line, possibly the use of (<) or (>), any ideas?

**Comment by CharlieRoss2506** [comment]
I don't think the video was working for anyone, not sure if its supported

**Comment by CharlieRoss2506** [comment]
> Im getting a Syntax-error for the %%HTML part of 5.4 that I'm unsure how to resolve, its specifically for the second line, possibly the use of (<) or (>), any ideas?

If you could still put the code in here for reference, tah :)

**Comment by charlotteharrison1** [comment]
> I don't think the video was working for anyone, not sure if its supported

yeah I've gotten the same thing even when I just run the code given in the notebook
